lib/rucio/web/ui/main.py

```diff
@@ -23,6 +23,13 @@
 from rucio.common.utils import generate_http_error
 from rucio.web.ui.common.utils import check_token, get_token
 
+import gzip
```

Comment:
Integrate the import with the already existing ones. e.g., parts of web are already imported before. The order of imports should be:

1. standard library imports
2. related third party imports
3. local application/library specific imports